metadata-ingestion/src/datahub/ingestion/source/dbt/dbt_common.py

```diff
@@ -1684,6 +1685,12 @@ def _create_dataset_properties_aspect(
     def get_external_url(self, node: DBTNode) -> Optional[str]:
         pass
 
+    @staticmethod
+    def _truncate_code(code: str, max_length: int) -> str:
+        if len(code) > max_length:
+            return code[:max_length] + "..."
```

Comment:
I guess max_length + 3 doesn't matter, right? Otherwise, the correct would be:
`return code[:max_length-3] + "..."`